blog.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="zxx" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <script src="https://kit.fontawesome.com/239552ee32.js" crossorigin="anonymous"></script>
+  <title>Adopt A Dog | Dogs</title>
+  <link href="https://fonts.googleapis.com/css?family=Josefin+Slab&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <header class="header-container">
+    <div class="header-box1">
+      <img src="images/adoptadog-logo.svg" alt="sitting dog next to text of Adopt A dog">
+      <p>Seatles's premiere dog adoption agency</p>
+    </div>
+    <div class="header-box2 checkout-page-link">
+      <i id="checkout-icon" class="fas fa-shopping-cart"> $123.56</i>
+      <a href="checkout.html">Checkout</a>
+    </div>
+  </header>
+
+  <div class="menu-bar">
+    <nav>
+      <ul>
+        <li><a href="index.html">Home /</a></li>
+        <li><a href="dogs.html">Dogs /</a></li>
+        <li><a href="blog.html">Blog</a></li>
+      </ul>
+    </nav>
+  </div>
+
+  <section class="blog-box-wrapper">
+    <h2 class="h-title">Adopt a dog Blog</h2>
+
+    <article class="blog1">
```

Comment:
You should reuse a class for each of these blogs, that way you don't have to create a new class for each blog, epecsially as they're the same